website/docs/r/branch_protection.html.markdown

```diff
@@ -78,6 +78,7 @@ The following arguments are supported:
 * `dismissal_teams`: (Optional) The list of team slugs with dismissal access.
   Always use `slug` of the team, **not** its name. Each team already **has** to have access to the repository.
 * `require_code_owner_reviews`: (Optional) Require an approved review in pull requests including files with a designated code owner. Defaults to `false`.
+* `required_approving_review_count`: (Optional) Require x number of approvals to satisfy branch protection requirements. If this is specified it must be a number between 1-6. This limitation is from the github side, see the upstream [documentation](https://developer.github.com/v3/repos/branches/#parameters-1) for more information.
```

Comment:
Could you update this to capitalize GitHub? I'd also suggest saying "This matches GitHub's API requirement" instead of "This limitation". Thanks! I was about to add this myself before I saw the PR :)

Reply:
sure thing, I forgot I had this open :wink: